js/ui/date_box/ui.date_box.base.js

```diff
@@ -347,10 +347,20 @@ const DateBox = DropDownEditor.inherit({
 
     _renderPopup: function() {
         this.callBase();
+        this._renderVisibilityChangeEvent();
         this._popup._wrapper().addClass(DATEBOX_WRAPPER_CLASS);
         this._renderPopupWrapper();
     },
 
+    _renderVisibilityChangeEvent: function() {
+        const renderOpenedStateHandler = () => {
+            this._strategy.renderOpenedState();
+        };
+
+        this._popup.on('shown', renderOpenedStateHandler);
```

Comment:
```
_popupShownHandler: function() {
    this.callBase();

    this._strategy.renderOpenedState();
}
```

The same code for the `_popupHiddenHandler`

Reply:
thx